src/modules/storage/LedgerStorage.ts

```diff
@@ -248,7 +248,7 @@ export class LedgerStorage extends Storages
                     [
                         block.header.height.value,
                         0,
-                        '0x0000000000000000',
+                        Hash.NULL,
```

Comment:
```suggestion
                        enroll.random_seed,
```
Isn't this right now?

Reply:
Yes, It was previously changed to random_seed, but Hash.NULL was left for a while.